lib/formatters/text.js

```diff
@@ -17,9 +17,7 @@
 function formatText(req, res, body) {
 
     // if body is null, default to empty string
-    var data = '';
-
-    data = body.toString();
+    var data = body == null ? '' : String(body);
```

Comment:
Great catch!

I like the `== null` style myself for both `null` and `undefined`, but it fails linting in this project.

Could you provide a set of tests for this change? Would like to see
1) A test for when `.toString` is not defined
2) A test for when body is no provided
3) A test for when body is a string